### ISSUE TITLE
Unable to start game

### ISSUE THREAD

**Issue by ghost** [diddlesnaps/gog-galaxy-wine#18]
Installation of the client worked flawlessly without any (known) issues; however, I'm unable to start a game. It downloaded the .exe installer. Is using the native Linux version of a game not supported? Does *everything* run inside wine? How do I start my games?

**Comment by Tarcaxoxide** [comment]
im having this issue 2 , its weird .
game fails to launch through the launcher but if i go to the directory and manually run it , it works fine.

**Comment by lucyllewy** [comment]
yes, everything must run in WINE for the games library to work. Unfortunately that means there are a lot of issues with many of the games, while others might work well.

**Comment by lucyllewy** [comment]
If a game in your library has a Linux version then I recommend you install and use that directly over playing through my snap. That does mean that things like achievements won't be tracked, though :-(

**Comment by reyquito** [comment]
I'm also having this issue with Neverwinter Nights 2 and The Withcher: Enhanced Edition :(

**Comment by lucyllewy** [comment]
Hi, can you please try the release in `candidate` to see if that helps alleviate the issues?

<details>
<summary>If you already have the snap installed...</summary>

```bash
sudo snap refresh --candidate gog-galaxy-wine
```

</details>

<details>
<summary>If you don't have the snap currently installed...</summary>

```bash
sudo snap install --candidate gog-galaxy-wine
```

</details>
